app/src/lib/stores/helpers/branch-pruner.ts

```diff
@@ -167,14 +210,16 @@ export class BranchPruner {
         continue
       }
 
-      const branchName = branch.canonicalRef.substr(branchRefPrefix.length)
+      if (options.deleteBranch) {
+        const branchName = branch.canonicalRef.substr(branchRefPrefix.length)
 
-      const isDeleted = await gitStore.performFailableOperation(() =>
-        deleteLocalBranch(this.repository, branchName)
-      )
+        const isDeleted = await gitStore.performFailableOperation(() =>
+          deleteLocalBranch(this.repository, branchName)
+        )
 
-      if (isDeleted) {
-        log.info(`Pruned branch ${branchName} (was ${branch.sha})`)
+        if (isDeleted) {
+          log.info(`Pruned branch ${branchName} (was ${branch.sha})`)
```

Comment:
I think we want to log a message if the branch _would have been_ deleted. This way, we can verify that the right branches are being tagged for deletion without having to rely on setting breakpoints and stepping through the code.

Reply:
Check out how 0e820db2004b2b482e9f77aadbad6991be8207ee feels